graphql_compiler/schema/sqlalchemy_schema.py

```diff
@@ -1,17 +1,111 @@
 # Copyright 2019-present Kensho Technologies, LLC.
 from collections import namedtuple
 
+from graphql.type.definition import GraphQLInterfaceType, GraphQLObjectType
+import six
+import sqlalchemy
+
+from . import is_vertex_field_name
+
+
+# Describes the intent to join two tables on equality of a column of theirs.
```

Comment:
"equality of column of theirs" makes it sound as if though the columns need to be named the same.

Reply:
Can you suggest a rewording? Can I use the phrase "foreign key" without implying a foreign key constraint must be there?

Reply:
How about "Describes the intent to join two tables using the specified columns."?